challenge-04/README.md

```diff
@@ -7,15 +7,53 @@ um único parâmetro como argumento. Essa função deve retornar `true` se o
 equivalente booleano para o valor passado no argumento for `true`, ou `false`
 para o contrário.
 */
-?
+var isThuthy = function(param) {
+	if(!!param) {
```

Comment:
-1
@adamsalves quando você usa o operador para obter o equivalente booleano de algum valor, isso já retorna `true` ou  `false`. Sabendo disso, dá pra reduzir o corpo dessa função para apenas uma única linha ;)

Entendeu o porquê?

Basicamente você está fazendo:

- Se o equivalente booleando de `param` for `true`, retorne `true`;
- Senão, retorne `false`.

Você só tem 2 tipos de valores para retornar, e esses valores já são retornados pelo operador `!!` ;)

Compreendeu?